web/src/recent_view_ui.js

```diff
@@ -932,6 +933,9 @@ export function show() {
     search.clear_search_form();
     complete_rerender();
     resize.update_recent_view_filters_height();
+    if (activity.user_filter) {
+        activity.build_user_sidebar();
+    }
```

Comment:
Similarly I think this goes in `pm_list.handle_narrow_deactivated`.

Reply:
Ah great, removed this from here